src/config/dynamic-theme-fixes.config

```diff
@@ -5792,6 +5792,15 @@ img[src="images/home_banner.png"]
 
 ================================
 
+openedu.ru
+
+CSS
+button:focus {
+    outline: none;
```

Comment:
If I'm allowed to give an input here (even though I'm not a maintainer of Dark Reader): This solution causes accessibility issues as the outline style is significant for keyboard navigation. The outline shouldn't be removed without adding a sufficient alternative, e.g.

```css
outline: 2px solid transparent; /* Windows High-Contrast Mode fix */
box-shadow: 0 0 0 2px whatevercolourfits; /* Replace default outline with box-shadow */
```